.github/workflows/integration-test-cli.yaml

```diff
@@ -0,0 +1,27 @@
+name: Integration Tests CLI
```

Comment:
Should this be the same as the name of the job or do we want this to be different?
```suggestion
name: CLI Integration Tests
```

Reply:
We can use the same name, I'll apply this suggestion.